src/impu_store.cpp

```diff
@@ -539,6 +539,40 @@ Store::Status ImpuStore::set_impu_without_cas(ImpuStore::Impu* impu,
   return status;
 }
 
+Store::Status ImpuStore::add_impu_without_cas(ImpuStore::Impu* impu,
```

Comment:
Possibly just `add_impu`? I don't think `add_impu_with_cas` would be meaningful.